content/_includes/page/base.njk

```diff
@@ -24,4 +24,10 @@
     {{ content | typogr | safe }}
     {% include "page/list.njk" %}
   </div>
+
+  <!-- used for webmentions -->
+  <a href="{{ page.url | url }}" class="u-url" hidden>{{ full_title | safe }}</a>
+  {% for link in syndication %}
+    <a rel="syndication" class="u-syndication" href="{{ link | url }}" hidden>{{ link | url }}</a>
+  {% endfor %}
```

Comment:
@mirisuzanne Is the first <a> on the page after the ` <!-- used for webmentions -->` here missing the title? This screenshot is from the deploy preview on Ed's article:  https://deploy-preview-218--oddleventy.netlify.app/2022/06/01/dependabot-single-pull-request/

![Screen Shot 2022-06-30 at 3 21 27 PM](https://user-images.githubusercontent.com/1581694/176772103-6dbc73e2-103c-49aa-8873-51c023293707.jpg)